ssl_exporter.go

```diff
@@ -119,6 +125,7 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 		subject_emails := cert.EmailAddresses
 		subject_ips := cert.IPAddresses
 		serial_no := cert.SerialNumber.String()
+		ous := cert.Subject.OrganizationalUnit
```

Comment:
Could you change this variable to `subject_ous`? Just for the sake of parity with other vars.